Message/AndroidMessage.php

```diff
@@ -52,11 +59,11 @@ class AndroidMessage implements MessageInterface
     protected $allIdentifiers = array();
 
     /**
-     * Options for GCM messages
+     * Options for FCM messages
      *
      * @var array
      */
-    protected $gcmOptions = array();
+    protected $fcmOptions = array();
 
     /**
      * Sets the string message
```

Comment:
Hello,
In the FCM case setMessage method will not work :) it should be replaced by body attribute. We can add also "title" attribute.

`public function setMessage($message)
{
	if ($this->isFCM){
		$this->body = $message;
	} else {
		$this->message = $message;
	}
}
`

Reply:
Ah! Good catch. Thanks. I'll get that fixed shortly.